### ISSUE TITLE
Need to handle German Sonderwurst

### ISSUE THREAD

**Issue by antrik** [Project60/org.project60.sepa#48]
So you thought that with the SEPA file format being an European standard, it would work across Europe?

Well, guess again. German banks decided the standard format ain't good enough for them, and came up with an own subset. This would be fine if it was a recommendation of what is most likely to work reliably -- but it's not. According to our initial testing, at least some German banks actually refuse to accept standard SEPA files, and will only take those using the German variant. Hurray for standards...

In other words, we will need some conditionals in the SDD extension, so it can also generate German SEPA files instead of standard SEPA files. Or we could perhaps introduce some hook(s) so an extra "German SEPA" extension could modify the behaviour of the standard SEPA extension as needed... Though the latter would bear a distinct tinge of over-engineering in my opinion :-)
